src/index.html

```diff
@@ -1,14 +1,56 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta charset="UTF-8" >
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no,
+      initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" >
     <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css?family=Roboto:500"
+    >
+    <link rel="stylesheet" href="./normalize.css" >
+    <link rel="stylesheet" href="./style.css" >
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="logo__link">
+        <img class="logo__image" src="./images/logo.png"
+        alt="logo">
```

Comment:
```suggestion
        <img 
          class="logo__image" 
          src="./images/logo.png"
          alt="logo"
        >
```
![image](https://github.com/mate-academy/layout_moyo-header/assets/96409563/a9b1e327-a057-41ba-9fbb-451ad4f26eb1)